### PR TITLE
maintenance, tox, github actions

### PR DESCRIPTION
**Pull request by JakaKokosar**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[tox]
+envlist =
+    py{37,38}-orange-{oldest, latest, released}
+    future
```
Comment (markotoplak) [inline tox.ini:4]: Maybe delete `future`

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    # our desire to have OrangeDeprecationWarnings raised
+    PYTHONWARNINGS=module
+    # set coverage output and project config
+    COVERAGE_FILE = {toxinidir}/.coverage
+    COVERAGE_RCFILE = {toxinidir}/.coveragerc
+deps =
+    pyqt5==5.12.*
+    pyqtwebengine==5.12.*
+    oldest: orange3==3.25.0
+    oldest: orange-canvas-core==0.1.15
```
Comment (markotoplak) [inline tox.ini:31]: Please move to oldest dependencies that still pass, and comment where these are not the same as for the then-released Orange

**Comment by markotoplak** [comment]
Thanks @JakaKokosar, I am trying this one out on some other repo, quasars/orange-spectroscopy#507.

I like the simplicity and thoroughness of your tox and github actions files a lot.

File: tox.ini (hunk 1 above)
Comment (JakaKokosar) [reply tox.ini]: whoops, thanks!

**Comment by JakaKokosar** [comment]
@markotoplak If tests pass I think this is OK to merge

**Pull request merged**